routes/events.js

```diff
@@ -89,4 +101,66 @@ router.delete(
   handleEventDelete
 );
 
+// Events projects requirements endpoints
+router.post(
```

Comment:
Use the `checkEventOwner` validator in the EventValidation class to ensure only the event owner/creator can access these routes.